runtime/compilers/rillv1/parse_component.go

```diff
@@ -17,6 +17,11 @@ var vegaLiteSpec string
 
 var vegaLiteSchema = jsonschema.MustCompileString("https://vega.github.io/schema/vega-lite/v5.json", vegaLiteSpec)
 
+//go:embed data/component-template-v1.json
+var componentTemplateSpec string
+
+var componentTemplateSchema = jsonschema.MustCompileString("https://github.com/rilldata/rill/runtime/compilers/rillv1/data/component-template-v1.json", componentTemplateSpec)
```

Comment:
Is there a way we can use a relative path instead of an URL. We might want to support Rill developer without network access as much as possible.

Reply:
It's confusing, but this doesn't actually make a network request. It's just JSON schema expecting a URL that identifies the particular schema (it seems to include it in some error messages). 

For example, we load the Vega spec like this a few lines above, but that also doesn't do a network request:
```go
var vegaLiteSchema = jsonschema.MustCompileString("https://vega.github.io/schema/vega-lite/v5.json", vegaLiteSpec)
```